infra/resources/modules/commons/containers.tf

```diff
@@ -45,3 +45,65 @@ resource "azurerm_cosmosdb_sql_container" "subscription_history" {
     }
   }
 }
+
+resource "azurerm_cosmosdb_sql_container" "activations" {
+  name                  = "activations"
+  resource_group_name   = azurerm_resource_group.data_rg.name
+  account_name          = module.cosmosdb_account.name
+  database_name         = module.cosmosdb_sql_database_trial.name
+  partition_key_path    = "/trialId"
+  partition_key_version = 2
+
+  autoscale_settings {
+    max_throughput = 1000
+  }
+
+  indexing_policy {
+    included_path {
+      path = "/*"
+    }
+
+    excluded_path {
+      path = "/\"_etag\"/?"
+    }
+
+    composite_index {
+      index {
+        path  = "/trialId"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/type"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/isActive"
+        order = "Ascending"
+      }
+
+      index {
+        path  = "/id"
+        order = "Descending"
```

Comment:
**question**: Why the order is Descending? Maybe I missed something, but I don't understand this choice. Assuming that this index is created to support the query to retrieve the requests to be activated, I would personally have set it to ascending since it follows the activation flow to be respected (from oldest to newest).

Reply:
You are right, it was a mistake. I agree it should be `Ascending`